website/scripts/heron-version.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+
```

Comment:
Please provide some human-speak comment that gives an overview of what this script does.